pipelines/DarkEnergyCamera/ApPipe.yaml

```diff
@@ -0,0 +1,35 @@
+description: End to end AP pipeline specialized for DECam
+# (1) Run $AP_PIPE_DIR/pipelines/DarkEnergyCamera/RunIsrForCrosstalkSources.yaml
+# (2) Execute `make_apdb.py` with the same configs used here for diaPipe, e.g.,
+#     make_apdb.py -c isolation_level=READ_UNCOMMITTED
+#                  -c db_url="sqlite:////project/user/association.db"
+# (3) Run this pipeline
```

Comment:
Is there any way to do this as a single pipeline? It's going to be a problem for `ApVerify.yaml` if there is no "the AP pipeline" for DECam.

Reply:
No 😞 Chris Waters can explain why better than I can. If you must have a single pipeline, the only alternative is to skip inter-chip crosstalk by using the DECam `RunIsr.yaml` as I mentioned above.

Reply:
😭